demo/server/keycloak.json

```diff
@@ -0,0 +1,9 @@
+{
+  "realm": "raincatcher",
+  "auth-server-url": "http://localhost:8080/auth",
```

Comment:
Actually we should may profile based configuration with different values for every environment lads will have.